mmv1/products/appengine/terraform.yaml

```diff
@@ -82,6 +82,27 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           service_name: "ae-service"
         test_env_vars:
           org_id: :ORG_ID
+      - !ruby/object:Provider::Terraform::Examples
+        name: "app_engine_standard_app_version_outbound"
+        primary_resource_id: "myapp_v3"
+        ignore_read_extra:
+          - "delete_service_on_destroy"
+        vars:
+          appengine_static_ip_network: "apg-static-ip-network"
+          appengine_static_ip: "apg-static-ip"
+          appengine_static_ip_conn: "apg-conn"
+          appengine_static_ip_router: "apg-static-ip-router"
+          appengine_static_ip_addr: "apg-static-ip-addr"
+          appengine_static_nat: "apg-static-nat"
+          appengine_static_ip_service: "apg-static-ip-service"
+          project_id: "ae-project"
+          bucket_name: "appengine-static-content"
+          service_name: "ae-service"
+        min_version: beta
+        skip_test: true
+        test_env_vars:
+          org_id: :ORG_ID
+        skip_docs: true
```

Comment:
Sorry to revise, but I notice the test is skipped and not in docs. Can we surface this example in docs? Otherwise this test and block can be removed. We no longer generate docs for external dependencies.

Reply:
I can show it in the documentation, it's a standard case to get app_engine streams out on a fixed ip. and it allows a provider to whitelist your IP. This example is a bit long but it's a nice example.
As you wish. I either post or delete. When do you think?

Reply:
Lets remove it or perhaps make the sample more documentation focused. We don't have a way to compress the example in the documentation so you could make it just refer to the really important bits. Briefly mentioning what else is set up. Up to you.

Reply:
@ScottSuarez ok i remove this example. Nevertheless, it allowed me to learn a lot of things. Thanks.